src/PAModel/Entropy.cs

```diff
@@ -58,6 +58,9 @@ internal class PropertyEntropy
         // Key is resource name, value is filename
         public Dictionary<string, string> LocalResourceFileNames { get; set; } = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
 
+        // Some of the properties have different value of InvariantScript than the DefaultScript which should be the case, hence need special handling.
+        public Dictionary<string, string[]> FunctionParamsInvariantScripts { get; set; } = new Dictionary<string, string[]>(StringComparer.OrdinalIgnoreCase);
```

Comment:
is string[] correct? Couldn't we just persist a dict of <propName, defaultScript>, since the invariant script is in the Yaml?

Reply:
Discussed offline, Sushil to add comment explaining why this is the case.